es/i18n/es.json

```diff
@@ -146,6 +146,41 @@
         "no-travel-hint": "Para ver más, cambia los filtros de arriba o registra algunos viajes!"
     },
 
+    "multilabel":{
+        "walk": "Caminando",
+        "e-bike": "e-bicicleta",
+        "bike": "Bicicleta",
+        "bikeshare": "Bicicleta compartida",
+        "scootershare": "Motoneta compartida",
+        "drove_alone": "Coche de Gas, Condujo solo",
+        "shared_ride": "Coche de Gas, Condujo con otros",
+        "e_car_drove_alone": "e-coche, Condujo solo",
+        "e_car_shared_ride": "e-coche, Condujo con ontras",
+        "moped": "Ciclomotor",
+        "taxi": "Taxi / Uber / Lyft",
+        "bus": "Autobús",
+        "train": "Tren",
+        "free_shuttle": "Colectivo gratuito",
+        "air": "Avión",
+        "not_a_trip": "No es un viaje",
+        "no_travel": "No viajar",
+        "home": "Inicio",
```

Comment:
I know this has been the translation for a while, but while we're making changes here, is there a reason we're using "inicio" instead of "casa" or even "residencia" here? I take "inicio" to mean "start" or "beginning" which is sort of the same idea, since home is usually where one starts off for the day, but I think "casa" is a much better translation for "home".

Reply:
```suggestion
        "home": "Casa",
```